src/components/Book.js

```diff
@@ -0,0 +1,10 @@
+/* eslint react/destructuring-assignment: 0 */
+/* eslint react/prop-types: 0 */
+const Book = (props) => (
+  <div>
+    {props.bookName}
```

Comment:
- [ ] It would be better if we rename the bookName to bookAuthor as required in the project specification

Reply:
sorry for the misinformation